core/src/avm2/call_stack.rs

```diff
@@ -0,0 +1,63 @@
+use crate::avm2::function::Executable;
+use crate::avm2::Error;
+use crate::string::WString;
+use gc_arena::{Collect, MutationContext};
+
+const MAX_CALLSTACK: usize = 64;
```

Comment:
To be sure, is it the same in AVM2? Feels pretty low.

Reply:
So basically I tested this by doing infinite recursion, and then counting the calls in the stack traceback in the error. However, after testing again I think there was a mistake in the test, because it seems like the callstack can actually be larger than 64, **but 64 is the maximum that can be displayed in a stack traceback**. 

To confirm this, I ran the following code:
```as
var counter = 0;
function t() {
	counter++;
	t()
}
try { 
	t();
} catch (e) {}

trace(counter);
```
Which gives me 8816, so I will change it to that.

Reply:
Although this test may not be accurate since it did not count the calls before it called frame1(), so I have to count that as well

Reply:
Okay so, as you mentioned to me on discord, this is actually because it is similar to the regular C callstack, and i'm pretty sure it's the same as javascript's callstack. I ended up deciding on 500, but one thing I noticed while testing is that we can actually make the callstack much larger on release builds due to better inlining.